releases/release-8.5.1.md

```diff
@@ -0,0 +1,103 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: Learn about the improvements and bug fixes in TiDB 8.5.1.
+---
+
+# TiDB 8.5.1 Release Notes
+
+Release date: January xx, 2025
+
+TiDB version: 8.5.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.1/production-deployment-using-tiup)
+
+## Operating system and platform requirement changes
+
+Starting from v8.5.1, TiDB resumes the support for CentOS Linux 7. If you plan to deploy TiDB v8.5 or upgrade your cluster to v8.5, ensure you use TiDB v8.5.1 or a later version.
+
+- According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 7 ends on June 30, 2024. In v8.4.0 DMR and v8.5.0, TiDB temporarily suspends the support for CentOS 7 and recommends you to use Rocky Linux 9.1 or later. Upgrading a TiDB cluster on CentOS 7 to v8.4.0 or v8.5.0 will cause the risk of cluster unavailability.
```

Comment:
I don't think we should say we temporarily suspended the support. Perhaps something like:

> TiDB v8.4.0 DMR and v8.5.0 releases dropped support and testing with CentOS Linux 7 due to it reaching [EOL status on June 30th, 2024](https://www.centos.org/centos-linux-eol/). To aid community users that are still running CentOS Linux 7, we resumed testing of CentOS Linux 7 with v8.5.1. TiDB continues to recommend that users migrate to a currently supported distribution for production usage.